src/packages.jl

```diff
@@ -87,11 +86,10 @@ function install(spec::String; interactive::Bool = true, pkgdir::AbstractString
 end
 
 """
-    update(spec::String; quiet::Bool = false)
+    update(spec::String; interactive::Bool = true, pkgdir::AbstractString = DEFAULT_PKGDIR, quiet::Bool = false)
```

Comment:
Perhaps write `GAP.Packages.DEFAULT_PKGDIR`, since the variable is not exported?